src/redux/sagas/expenditures/createExpenditure.ts

```diff
@@ -66,7 +67,12 @@ function* createExpenditure({
       context: ClientType.ColonyClient,
       methodName: 'makeExpenditure',
       identifier: colonyAddress,
-      params: [1, BigNumber.from(2).pow(256).sub(1), 1],
+      params: [
+        // @TODO: Get the permissions domain id using colony-js's getPermissionProofs
```

Comment:
Stuff like this _(`withProofs` helpers)_, is what I think needs to be removed from `colonyJS` and brought in the CDapp as contract helpers